.github/pull_request_template.md

```diff
@@ -0,0 +1,7 @@
+## 🔹 Jira Ticket(s) if any
+
+* [JIRA-XXXX](https://iterable.atlassian.net/browse/JIRA-XXXX)
```

Comment:
Suggestion: Change the ticket prefix to `MOB` since that will be the default ticket type we review